### ISSUE TITLE
non-game chat apps indicate "multiplayer"

### ISSUE THREAD

**Issue by danrabbit** [elementary/appcenter#1676]
## What Happened

![Screenshot from 2021-09-15 12 27 49](https://user-images.githubusercontent.com/7277719/133497173-eb872d27-7c43-4052-9976-659aacef590c.png)

## Expected Behavior

non-games shouldn't say "multiplayer". We should check app categories before showing this

**Comment by cassidyjames** [comment]
@danrabbit hm, I kind of wonder if we can combine multiplayer under online interactions?

so if `social-chat == MILD && social-audio == NONE`, we show something like

>**Online Interactions**
>Interaction with other people online without any kind of chat

And then keep the existing two:

>**Online Interactions**
>Moderated audio, video, or text messaging with other people

or 

>**Online Interactions**
>Unmoderated audio, video, or text messaging with other people

depending on the `social-audio` value

**Comment by cassidyjames** [comment]
I guess that's unless you're wanting to consider the "multiplayer" flag as being useful aside from just content warnings. Like if it's handy to explicitly point out multiplayer games.

**Comment by danrabbit** [comment]
Yeah, I think I'd still like to call it "Multiplayer" since it's not really interacting outside of gameplay, but I can see how the presence of chat implies multiplayer as well. I kinda just don't like how these tags are combined like this in OARS tbh
